docs/api-docs.yaml

```diff
@@ -1,3320 +1,3348 @@
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  message:
+                    type: string
+                    example: Job level deleted successfully.
+          
+        '401':
+            description: Unauthorized
+            content:
+              application/json:
+                schema:
+                  type: object
+                  properties:
+                   message:
+                      type: string
+                      example: "Unauthorized - Invalid credentials"            
+        '403':
+          description: Forbidden - User does not have permission
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  message:
+                    type: string
+                    example: Forbidden - User does not have permission 
+        '404':
+          description: Job level not found
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  message:
+                    type: string
+                    example: Job level not found
+  # /jobs/{jobId}/analytics:
+  #   get:
+  #     tags:
+  #       - job
+  #     summary: Get job analytics by Id
+  #     description: Returns detailed analytics data for a specific job.Must have ADMIN authority.
+  #     security: 
+  #     - jwt: []
+  #     operationId: getJobAnalytics
+  #     parameters:
+  #       - name: jobId
+  #         in: path
+  #         description: Id of the job to get analytics for
+  #         required: true
+  #         schema:
+  #           type: integer
+  #           format: int64
+  #       - name: startDate
+  #         in: query
+  #         description: Start date for analytics (format YYYY-MM-DD)
+  #         schema:
+  #           type: string
+  #           format: date
+  #       - name: endDate
+  #         in: query
+  #         description: End date for analytics (format YYYY-MM-DD)
+  #         schema:
+  #           type: string
+  #           format: date
+  #     responses:
+  #       '200':
+  #         description: Successful operation
+  #         content:
+  #           application/json:
+  #             schema:
+  #               $ref: '#/components/schemas/JobViews'
+  #       '400':
+  #         description: Invalid Id supplied
+  #       '404':
+  #         description: Job not found   
+  /jobs/{id}/views:
+    get:
+      tags:
+        - job
+      summary: Get view count for a job
+      parameters:
+        - name: id
+          in: path
+          description: ID of resource
+          required: true
+          schema:
+            type: integer
+      responses:
+        '200':
+          description: success
+          content:
+            application/json:
+              schema:
+                  type: object

```

Comment:
Wrap your response with the data object. Check this example: https://develeb.github.io/Develeb-Platform/#/job/getJobById

Also rename resource_Id to job_id